test/e2e/storage/csi_objects.go

```diff
@@ -37,12 +38,30 @@ import (
 	. "github.com/onsi/ginkgo"
 )
 
-const (
-	csiHostPathPluginImage      string = "quay.io/k8scsi/hostpathplugin:v0.2.0"
-	csiExternalAttacherImage    string = "quay.io/k8scsi/csi-attacher:v0.2.0"
-	csiExternalProvisionerImage string = "quay.io/k8scsi/csi-provisioner:v0.2.1"
-	csiDriverRegistrarImage     string = "quay.io/k8scsi/driver-registrar:v0.2.0"
-)
+var csiImageVersions = map[string]string{
+	"hostpathplugin":   "v0.2.0",
+	"csi-attacher":     "v0.2.0",
+	"csi-provisioner":  "v0.2.1",
+	"driver-registrar": "v0.2.0",
+}
+var csiImageVersion string
+var csiImageRegistry string
+
+func init() {
```

Comment:
Should these be defined in framework/test_context.go instead?

Reply:
I didn't know that there was some centralized argument handling. As this seems to be preferred, I'll update the PR to put the flags there.

Reply:
Done, see the separate commit cc2a5954acf570b7f919a9a7e274661cc830d3a3. I can squash if you want.